aioftp/server.py

```diff
@@ -857,6 +857,11 @@ def __init__(self,
             loop=self.loop
         )
         response_queue = asyncio.Queue(loop=self.loop)
+        if hasattr(asyncio, 'current_task'):
```

Comment:
Maybe:
```python
IS_PY37 = sys.version_info[:2] == (3, 7)
current_task = asyncio.current_task if IS_PY37 else asyncio.Task.current_task
```

Reply:
Changed... (and not sure on the best place for it admittedly)